### PR TITLE
Updated rust coverage tools evaluation

### PR DESCRIPTION
**Pull request by alekseyborisyukvalidas**
- Update malfunction model
- Added qualification section

### CODE REVIEW

File: docs/score_tools/tools_static_analysis_code_quality/symbol_report_blanket.rst [hunk 1]
```diff
@@ -85,62 +85,61 @@ qualified and output of coverage data in `.profraw` format is correct. Due to th
...
+     - | Reported higher statement/branch coverage than in coverage data
+       |
+       | The tool outputs statement or branch coverage values that are greater than those found in the raw coverage data.
      - yes
-     - Since we want to achieve 100% branch coverage (`check here <https://eclipse-score.github.io/score/main/platform_management_plan/software_verification.html#quality-criteria>`_) this would stand out and be manually investigated.
      - no
      - no
-     - high
+     - yes
+     - low
```
Comment (pawelrutkaq) [inline docs/score_tools/tools_static_analysis_code_quality/symbol_report_blanket.rst:107]: Can't this be elevated to high by providing a release test suite for reporting test cases based on edge case and fuzzy tests ? Why this then requires imidie qualification and not a mitigation technique and an additional judgment of likelihood based on scope of usage on codebases like `libcore`.

File: docs/score_tools/tools_static_analysis_code_quality/symbol_report_blanket.rst [hunk 2]
```diff
@@ -85,62 +85,61 @@ qualified and output of coverage data in `.profraw` format is correct. Due to th
...
-     - Coverage reported for another function
-     - Wrong reporting view
-     - no
-     - | Manual review would disclose that coverage is reported for another function than tested one by developer.
+     - Manual review of coverage report.
      - yes
      - no
      - high
 
+.. [*] Expected that correctness of function calls are tested via mocking (e.g. gtest macro EXPECT_CALL()).
```
Comment (pawelrutkaq) [inline docs/score_tools/tools_static_analysis_code_quality/symbol_report_blanket.rst]: do not refer to cpp in rust

File: docs/score_tools/tools_static_analysis_code_quality/symbol_report_blanket.rst (hunk 1 above)
Comment (alekseyborisyukvalidas) [reply docs/score_tools/tools_static_analysis_code_quality/symbol_report_blanket.rst:107]: This will be hard to argue. Mitigation usually is something that is done during every call of the tool, e.g. review of the output, inspection of the log etc.


Also, "test suite" is actually what "validation of software tool" qualification method of ISO suggest.

File: docs/score_tools/tools_static_analysis_code_quality/symbol_report_blanket.rst (hunk 2 above)
Comment (alekseyborisyukvalidas) [reply docs/score_tools/tools_static_analysis_code_quality/symbol_report_blanket.rst]: removed in 2210ef9

**Comment by masc2023** [comment]
@pawelrutkaq , can you agree for now, if there is new information, we can change it anyway
